lib/datadog/core/configuration/components.rb

```diff
@@ -123,6 +125,7 @@ def initialize(settings)
           @health_metrics = self.class.build_health_metrics(settings, @logger)
           @appsec = Datadog::AppSec::Component.build_appsec_component(settings, telemetry: telemetry)
           @dynamic_instrumentation = Datadog::DI::Component.build(settings, agent_settings, @logger, telemetry: telemetry)
+          @error_tracking = Datadog::ErrorTracking::Component.build(settings, tracer, logger)
```

Comment:
It's better to stick with instance variable here

```suggestion
          @error_tracking = Datadog::ErrorTracking::Component.build(settings, @tracer, @logger)
```

Reply:
addressed in e5fe33bee6f4993593a80e133f14416eee497861